vgen/src/test/java/io/gapi/vgen/testdata/nodejs_discovery_fragments_appengine.v1beta4.json_fragments[nodejs_discovery_fragments_appengine.v1beta4.json].baseline

```diff
@@ -0,0 +1,397 @@
+
+(new GoogleAuth).getApplicationDefault(function(err, authClient) {
+  if (err === null) {
+    if (authClient.createScopedRequired && authClient.createScopedRequired()) {
+      var scopes = ['https://www.googleapis.com/auth/cloud-platform'];
+      authClient = authClient.createScoped(scopes);
+    }
+
+    var request = {
+      auth: authClient
+    };
+
+    // TODO: Change placeholders below to values for parameters to the 'get' method:
+
+    // * Part of `name`. Name of the application to get. For example: "apps/myapp".
+    request.appsId = "";
+
+    appengine.apps.get(request, function(error, result) {
+      if (error) {
+        console.log(error);
+      } else {
+        console.log(result);
+      }
+    });
+   }
+});
+var google = require('googleapis');
+var appengine = google.appengine('v1beta4');
+var GoogleAuth = require('google-auth-library');
+
+(new GoogleAuth).getApplicationDefault(function(err, authClient) {
+  if (err === null) {
+    if (authClient.createScopedRequired && authClient.createScopedRequired()) {
+      var scopes = ['https://www.googleapis.com/auth/cloud-platform'];
+      authClient = authClient.createScoped(scopes);
+    }
+
+    var request = {
+      auth: authClient
+    };
+
+    // TODO: Change placeholders below to values for parameters to the 'delete' method:
+
+    // * Part of `name`. Name of the resource requested. For example: "apps/myapp/modules/default".
+    request.appsId = "";
+    // * Part of `name`. See documentation of `appsId`.
+    request.modulesId = "";
+
+    appengine.apps.modules.delete(request, function(error, result) {
+      if (error) {
+        console.log(error);
+      } else {
+        console.log(result);
+      }
+    });
+   }
+});
+var google = require('googleapis');
+var appengine = google.appengine('v1beta4');
+var GoogleAuth = require('google-auth-library');
+
+(new GoogleAuth).getApplicationDefault(function(err, authClient) {
+  if (err === null) {
+    if (authClient.createScopedRequired && authClient.createScopedRequired()) {
+      var scopes = ['https://www.googleapis.com/auth/cloud-platform'];
+      authClient = authClient.createScoped(scopes);
+    }
+
+    var request = {
+      auth: authClient
+    };
+
+    // TODO: Change placeholders below to values for parameters to the 'get' method:
+
+    // * Part of `name`. Name of the resource requested. For example: "apps/myapp/modules/default".
+    request.appsId = "";
+    // * Part of `name`. See documentation of `appsId`.
+    request.modulesId = "";
+
+    appengine.apps.modules.get(request, function(error, result) {
+      if (error) {
+        console.log(error);
+      } else {
+        console.log(result);
+      }
+    });
+   }
+});
+var google = require('googleapis');
+var appengine = google.appengine('v1beta4');
+var GoogleAuth = require('google-auth-library');
+
+(new GoogleAuth).getApplicationDefault(function(err, authClient) {
+  if (err === null) {
```

Comment:
Codegen improvement.  Here, a short error handling block is idiomatic
```
if (err) {
   console.log('Authentication failed because of ', err);
   return;
}
```